### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by purcell**
In connection with https://github.com/melpa/melpa/pull/7516

### CODE REVIEW

File: rails-routes.el [hunk 1]
```diff
@@ -68,38 +83,44 @@
...
 (defun rails-routes--guess-route (controller-full-path)
-  "Try to get the route name when rails routes don't show to us. CONTROLLER-FULL-PATH:  controller name plus action."
+  "Guess the route name from CONTROLLER-FULL-PATH.
+CONTROLLER-FULL-PATH is the controller name plus action."
   (let ((controller-path (nth 0 (split-string controller-full-path "#"))))
     (replace-regexp-in-string "\/" "_" controller-path)))
 
-(defun rails-routes--find (INSERT-CLASS)
-  "Ask for the route you want and insert on code.  INSERT-CLASS: if t, insert the prefix class."
+(defun rails-routes--insert (insert-class)
```
Comment (otavioschwanck) [inline rails-routes.el:117]: You changed the name but not changed the places that is called.

Comment (otavioschwanck) [inline rails-routes.el:117]: the name of the parameter is different than the used on the body.  Change it to insert-CLASS.

**Review by otavioschwanck — CHANGES_REQUESTED** [review]
Thanks for your contribution, just fix the two problems that i commented.

File: rails-routes.el (hunk 1 above)
Comment (purcell) [reply rails-routes.el:117]: Yep, missed that. Fixed now.

Comment (purcell) [reply rails-routes.el:117]: As a tip, you can suggest inline fixes using the suggestion button, then apply them directly.

File: rails-routes.el [hunk 2]
```diff
@@ -108,45 +129,46 @@
...
-            (rails-routes--guess-route (nth 3 selected-value))) rails-routes-insert-after-path))
+            (rails-routes--guess-route (nth 3 selected-value)))
+          rails-routes-insert-after-path))
 
 ;;;###autoload
-(defun rails-routes-find ()
+(defun rails-routes-insert ()
   "Find rails routes on current project."
   (interactive)
   (rails-routes--find nil))
```
Comment (otavioschwanck) [inline rails-routes.el]:
```suggestion
  (rails-routes--insert nil))
```

File: rails-routes.el [hunk 3]
```diff
@@ -108,45 +129,46 @@
...
 
 ;;;###autoload
-(defun rails-routes-find-with-class ()
+(defun rails-routes-insert-with-class ()
   "Find rails routes on current project.  Also insert a prefix class.  This can be used outside views."
   (interactive)
   (rails-routes--find t))
```
Comment (otavioschwanck) [inline rails-routes.el]:
```suggestion
  (rails-routes--insert t))
```

**Review by otavioschwanck — CHANGES_REQUESTED** [review]

**Comment by otavioschwanck** [comment]
added two more comments @purcell

**Pull request merged**